config.go

```diff
@@ -167,6 +169,19 @@ func loadConfig() (*ConfigFlags, error) {
 		return nil, err
 	}
 
+	// Manually enforce testnet 11 net params so we do not have to 
+	// support this special network in kaspad.
+	if activeConfig.NetSuffix != 0 && !activeConfig.Testnet {
+		return nil, errors.New("The net suffix can only be used with testnet")
+	}
+	if activeConfig.NetSuffix != 11 && activeConfig.Testnet {
```

Comment:
Seems like this will fail also if `NetSuffix == 0`. Shouldn't it be `if activeConfig.NetSuffix != 0 && activeConfig.NetSuffix != 11 && activeConfig.Testnet` ?

Reply:
Oops! Committed from wrong computer! This had been refactored later.